homeassistant/components/rfxtrx.py

```diff
@@ -144,6 +145,8 @@ def _valid_light_switch(value):
         vol.Required(ATTR_DEVICE): cv.string,
         vol.Optional(ATTR_DEBUG, default=False): cv.boolean,
         vol.Optional(ATTR_DUMMY, default=False): cv.boolean,
+        vol.Optional(ATTR_RECV_MODES, default=None):
+            vol.All(vol.All(cv.ensure_list, [cv.string]), vol.Length(min=1))
```

Comment:
Do you have a list of valid modes?
So we can check that the modes are valid.

Reply:
The modes names are in pyRFXTRX. I will see how I can validate the list from there.

Reply:
Done!